component/prometheus/operator/types.go

```diff
@@ -21,6 +21,12 @@ type Arguments struct {
 	// Namespaces to search for monitor resources. Empty implies All namespaces
 	Namespaces []string `river:"namespaces,attr,optional"`
 
+	// ScrapeInterval is the default interval to scrape targets.
+	ScrapeInterval time.Duration `river:"scrape_interval,attr,optional"`
```

Comment:
I'd like to call this `DefaultScrapeInterval` and `DefaultScrapeTimeout`.

Reply:
I think it might make sense to make this an additional struct type called `scrape`, and include it as a block.

I will shortly be adding other things from the `Prometheus` crd like enforced honorLabels or limits, so there will be more stuff in there.

So it will be something like:

```
scrape {
  default_scrape_interval = "15s"
}
```

Then you can just pass that whole thing to the manager stuff.

Reply:
Addressed in 36ab5ccdb2ccc566e4c4c4effa0775d8da41adb1